ITEST/server-core/src/org/openl/itest/core/JettyServer.java

```diff
@@ -31,10 +31,10 @@ private JettyServer(String explodedWar, boolean sharedClassloader) {
         WebAppContext webAppContext = new WebAppContext();
         webAppContext.setResourceBase(explodedWar);
         webAppContext.setExtraClasspath(getExtraClasspath());
-        webAppContext.setAttribute("org.eclipse.jetty.server.webapp.WebInfIncludeJarPattern", ".*/classes/.*");
+        webAppContext.setAttribute("org.eclipse.jetty.server.webapp.WebInfIncludeJarPattern",
+            ".*/classes/.*|.*ruleservice.ws[^/]*\\.jar$");

```

Comment:
why do you need ```.*ruleservice.ws[^/]*\\.jar$```?

Reply:
The OpenLTracingFilter is packaged into a jar named org.openl.rules.ruleservice.ws.tracing.jaeger and this jar should be added for scanning by WebInfIncludeJarPattern, so I've added this pattern to include this jar into the context. So, the pattern isn't hardcoded for one specific jar, but all the jars are related to the rule services. Should it be limited to this one jar?

Reply:
shouldn't it be included in ws-all?

Reply:
This jar is included in ws-all.war file as other jars. 
The context configuration files like CorsFilter, CXFServlet, SpringInitializer, etc. are located at the WEB-INF/classes/... folder of extracted war file, so they're included by .*/classes/.* part, but the configuration class like OpenLTracingFilter which is located at the jar, which is placed in /lib folder isn't included automatically.